magento/pwa-studio
# Increase CartPage test coverage

## Description

Increase test coverage for talons in `peregrine/lib/talons/CartPage`

## Related Issue

Closes PWA-995

## Acceptance

Any developer

### Verification Stakeholders

Any developer

### Specification



### Verification Steps

1. Run: `yarn test --collectCoverageFrom=**/lib/talons/CartPage/**/*.js packages/peregrine/lib/talons/CartPage`
1. Verify tests succeed
1. Verify acceptable test coverage

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
